hat/api_import/models.py

```diff
@@ -8,6 +8,7 @@
     ("orgUnit", "Org Unit"),
     ("instance", "Form instance"),
     ("bulk", "Bulk Org Units and Instances"),
+    ("storageLog", "Storage logs"),
```

Comment:
What is this for?

Reply:
The `@safe_api_import()` decorator will create a new record of `APIImport`. So I just added this new type, it's the first argument of the decorator function.